MyCryptoHQ/MyCrypto
# Disable tabs by network configuration

Closes #1721

### Description

Gives the possibility of disable navigation tabs that don't make sense in a certain network.

### Changes

* Addition of new parameter `unsupportedTabs` in network configuration. 
* Addition of new prop `isNotEnabled` in NavigationLink (this prop determines the rendering of the component).

### Steps to Test

1. Choose a network config and add, for example, `unsupportedTabs: [TAB.ENS, TAB.SWAP]` in file `staticNetworks.ts`
2. Select that network
3. Check visually that this tabs are not displayed
